### ISSUE TITLE
Integrate Bootstrap Notify

### ISSUE THREAD

**Issue by viarocket** [viarocket/travelhalal#4]
I have integrated BootstrapNotify.js already couple of days ago. 

Can you integrate it further? (i.e. rail's flash and notice, including devise's too)

**Comment by beenavaghani74** [comment]
Yes. I can setup rail's alert, error, flash and notice (include devise)

**Comment by viarocket** [comment]
noted.
